src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -350,7 +407,8 @@ export class NotebookService extends Disposable implements INotebookService {
 		if (!notebookUri) {
 			return undefined;
 		}
-		let uriString = notebookUri.toString();
+		// Format the URI string without query or fragment
```

Comment:
I think the code is clear enough of what it's doing that this comment is unnecessary. Instead I would have the comment say WHY you're doing this.